### ISSUE TITLE
[BUG] Link missing for launch plan node in dynamic task

### ISSUE THREAD

**Issue by wild-endeavor** [lyft/flyte#547]
**Describe the bug**
If you have a dynamic task that yields a launch plan node runs correctly, but there's no link to the launched execution.

The information is there.  If you look at 
https://flyte.lyft.net/api/v1/node_executions/project_name/development/main_workflow_execution_id/f16jfkky?limit=1000
it shows
```
"workflow_node_metadata": {
      "executionId": {
        "project": "project_name",
        "domain": "development",
        "name": "fi6uzb3a"
      }
    }
```
(where `f16jfkky` is the launch plan node generated by the dynamic task)

Ask me for internal links if in Lyft.

**Expected behavior**
I just want to see the link to the fi6uzb3a when I click on the f16jfkky node.

**Flyte component**
- [ ] Overall
- [ ] Flyte Setup and Installation scripts
- [ ] Flyte Documentation
- [ ] Flyte communication (slack/email etc)
- [ ] FlytePropeller
- [ ] FlyteIDL (Flyte specification language)
- [ ] Flytekit (Python SDK)
- [x] FlyteAdmin (Control Plane service)
- [ ] FlytePlugins
- [ ] DataCatalog 
- [ ] FlyteStdlib (common libraries)
- [x] FlyteConsole (UI)
- [ ] Other

**To Reproduce**
Run a dynamically launched launch plan.

**Environment**
**Flyte component**
- [ ] Sandbox (local or on one machine)
- [ ] Cloud hosted
     * [ ] AWS
     * [ ] GCP
     * [ ] Azure
- [ ] Baremetal
- [ ] Other

**Comment by wild-endeavor** [comment]
cc @schottra @katrogan @kumare3

**Comment by katrogan** [comment]
we should already store this relationship in the db: https://github.com/lyft/flyteadmin/blob/master/pkg/repositories/models/execution.go#L48

**Comment by wild-endeavor** [comment]
Okay, also looked at the Admin API for the child execution:

https://flyte.lyft.net/api/v1/executions/project_name/development/fmc5kqry shows
```
"parent_node_execution": {
        "node_id": "f6cavb6q",
        "execution_id": {
          "project": "project_name",
          "domain": "development",
          "name": "qt11m0ycbg"
        }
      },
```
which is correct.

And the node execution API shows the correct metadata as in the description.  I think everything is correct between admin and propeller, the link just needs to show up somehow.  We may have to change something in the backend for it, but I think this is mostly a Console change at least?  Randy - do you agree?

**Comment by kumare3** [comment]
I think one issue is that somehow the console is showing the type for the sub node as DynamicTask, when actually the node is workflow node. Interestingly the sublisting feature works great

**Comment by schottra** [comment]
fixed by lyft/flyteconsole#131
included in flyteconsole v0.18.0
